Package.swift

```diff
@@ -12,9 +12,9 @@ let package = Package(
         )
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-syntax", exact: "509.0.0"),
+        .package(url: "https://github.com/apple/swift-syntax", "509.0.0"..<"999.0.0"),
         .package(url: "https://github.com/apple/swift-collections", from: "1.0.4"),
-        .package(url: "https://github.com/omochi/CodegenKit", branch: "main")
+        .package(url: "https://github.com/omochi/CodegenKit.git", from: "1.4.1"),
```

Comment:
`.git` を付けるかどうかは他の 2つと統一したいです。
今は付けないほうに寄せてます。

```suggestion
        .package(url: "https://github.com/omochi/CodegenKit", from: "1.4.1"),
```

最近どこかで `.git` 付ける方がいいという話を見たけど
やるなら別のPRにして理由を整理しておきたいです。